From 6f04f02166b4f7876d69593af1c93872385edb13 Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Thu, 13 Sep 2012 09:40:24 -0700
Subject: [PATCH 140/795] MR1: CLOUDERA-BUILD. Updating build.xml's version to be consistent.
 (cherry picked from commit 4d624b58457c7a51a670bc1e34cc9f7d12456281)

(cherry picked from commit fbad4093b137121fad196f70fc25e994b0524b78)
---
 build.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/build.xml b/build.xml
index 9683fa9..ceff65a 100644
--- a/build.xml
+++ b/build.xml
@@ -54,7 +54,7 @@
  
   <property name="Name" value="Hadoop"/>
   <property name="name" value="hadoop"/>
-  <property name="version" value="0.20.2-cdh4b2-SNAPSHOT"/>
+  <property name="version" value="2.0.0-mr1-cdh4.2.0-SNAPSHOT"/>
   <property name="final.name" value="${name}-${version}"/>
   <property name="test.final.name" value="${name}-test-${version}"/>
   <property name="year" value="2009"/>
-- 
1.7.0.4

